src/run.rs

```diff
@@ -670,6 +671,15 @@ fn vacuum(
         });
     }
 
+    // If the user provided the `--older-than` argument, we need to filter out images which are newer than the provided duration.
+    if let Some(duration) = older_than {
+        let now = SystemTime::now();
+        let time_stamp = (now - *duration).duration_since(UNIX_EPOCH).unwrap();
```

Comment:
Our code style is that all usages of `unwrap` should be justified by a short comment explaining how it's not going to fail (search the code for `unwrap` for examples), or in what (rare) situations it may fail.

Reply:
I've implemented a match instead of the unwrap to handle the errors that can occur